raiden/network/proxies/token_network.py

```diff
@@ -900,31 +871,6 @@ def _check_why_deposit_failed(
 
         return error_type, msg
 
-    def _close_preconditions(
-            self,
-            channel_identifier: ChannelID,
-            partner: Address,
-            block_identifier: BlockSpecification,
-    ):
-        if not self.client.can_query_state_for_block(block_identifier):
```

Comment:
Again, this was sucestible to the same race condition as https://github.com/raiden-network/raiden/pull/3897/files#r278028913